pyproject.toml

```diff
@@ -64,7 +64,6 @@ altair = { version = "*", optional = true }
 asyncmy = { version = "*", optional = true }
 asyncpg = { version = "*", optional = true }
 pympler = { version = "*", optional = true }
-aiofiles = { version = "*", optional = true }
```

Comment:
I think we should return async file dbs but with async lock guarding every db method.
That way we get atomic operations without blocking other async processes.